### PR TITLE
Added gradle build compatibility to picard public. This requires a me…

### PR DESCRIPTION
**Pull request by jacarey**
…rge of the gradle changes in htsjdk

### CODE REVIEW

**Comment by jacarey** [comment]
Failures are expected until htsjdk change is merged.

File: build.xml [hunk 1]
```diff
@@ -132,6 +136,18 @@
         </loadfile>
     </target>
 
+    <target name="write-version-property">
+    	<exec outputproperty="htsjdk-version" executable="${gradle.executable}" dir="${htsjdk}">
+    		<arg value="-q"/>
+			<arg value="printVersion" />
+		</exec>
```
Comment (nh13) [inline build.xml]: indentation?

File: build.xml [hunk 2]
```diff
@@ -156,29 +172,36 @@
     </target>
 
     <target name="compile-htsjdk" depends="set-htsjdk-version" description="Build HTS-JDK, and grab build jars">
-        <ant antfile="build.xml" dir="${htsjdk}" target="clean" inheritall="false"/>
-        <ant antfile="build.xml" dir="${htsjdk}" target="all" inheritall="false"/>
+		<exec executable="${gradle.executable}" dir="${htsjdk}">
+			<arg value="assemble" />
+		</exec>
```
Comment (nh13) [inline build.xml]: indentation?

File: build.xml [hunk 3]
```diff
@@ -156,29 +172,36 @@
...
         </copy>
-        <!-- set the htsjdk version -->
-        <ant antfile="build.xml" dir="${htsjdk}" target="write-version-property" inheritall="false"/>
     </target>
 
     <target name="clean-htsjdk" description="Clean HTS-JDK">
-        <ant antfile="build.xml" dir="${htsjdk}" target="clean" inheritall="false"/>
+        <exec executable="${gradle.executable}" dir="${htsjdk}">
+			<arg value="clean" />
+		</exec>
```
Comment (nh13) [inline build.xml]: indentation?

Comment (nh13) [inline build.xml]: there's more below so I'll stop now.

**Pull request merged**